src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1015.UriMustBeAbsolute_Valid.sarif

```diff
@@ -1,38 +1,39 @@
 {
-  "$schema": "http://json.schemastore.org/sarif-2.0.0",
-  "version": "2.0.0",
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.4.json",
+  "version": "2.1.0",
   "runs": [
     {
       "tool": {
-        "name": "CodeScanner",
-        "downloadUri": "http://www.example.com/tools/codescanner/download.html"
+        "driver": {
+          "name": "CodeScanner",
+          "downloadUri": "http://www.example.com/tools/codescanner/download.html",
+          "rules": [
+            {
+              "id": "TST0001",
+              "helpUri": "http://www.example.com/rules/tst0001.html"
+            }
+          ]
+        }
       },
+      "versionControlProvenance": [
+        {
+          "repositoryUri": "https://example.com/my-project"
+        }
+      ],
       "results": [
         {
-          "message": { "text": "Some testing occurred." },
           "ruleId": "TST0001",
+          "ruleIndex": 0,
           "level": "error",
+          "message": {
+            "text": "Some testing occurred."
+          },
           "workItemUris": [
             "https://example.com/my-project/issues/42"
           ]
         }
       ],
-      "originalUriBaseIds": {
-        "SRCROOT": "file:///c:/Code/sarif-sdk/src"
-      },
```

Comment:
The important change is that the SARIF1015 test files no longer have `run.originalUriBaseIds`, because that is tested by the new SARIF1018.